Adds "FAQs" page to Trio Docs

Natolida opened the pull request:
This PR replaces PR #81  and adds a FAQ page to Trio Docs. 
- I have updated the syntax of the Tips, Warning, etc, with the recent change to MkDocs.
- I have confirmed that all links should be correct.

Natolida committed:
Create faq.md

- Adds faq.md to resources folder

Natolida committed:
Update mkdocs.yml

-Adds FAQ to Navigation

Natolida committed:
Update faq.md

- Adjust some content to MkDocs formatting
- Updates links

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+No. Trio lets you move between different pumps and CGM configurations within the app. 
+
+### I am moving from another closed-loop system. Will my settings transfer over?
+If you've been using other closed-loop solutions, it's essential to understand the differences between the one you're used to and Trio. Before making the transition, we recommend that you take the time to read through the documentation to build a basic understanding of Trio. Detailed information on transitioning from other closed-loop systems can be found [here](.../Configuration/transition-qa.md ...
```
There is an extra dot in the URL which should be:

```markdown
[here](../Configuration/transition-qa.md)
```

Natolida replied:
Corrected

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+Following the release of version 3.0.0, the project’s direction significantly changed due to differing views on development, open source, and peer review. This led to the separation from the iAPS repository and the birth of Trio as a distinct entity. This transition marks the project’s new phase, symbolizing its evolution and the collaborative development’s dynamic nature.
+
+!!! important "Important"
+_Trio is a DIY, open-source community project. As such, the user must build and maintain their app with the support of the Trio community. Trio has not undergone evaluation by any regulatory agency and is for use at your own risk._ 
```
In this file, each line of the **body text** of each admonition, must be indented 4-spaces, like this:

```markdown
!!! important "Important"
    _Trio is a DIY, open-source community project. As such, the user must build and maintain their app with the support of the Trio community. Trio has not undergone evaluation by any regulatory agency and is for use at your own risk._ 
```

It currently looks like this:

> ![CleanShot 2024-11-13 at 21 32 19@2x](https://github.com/user-attachments/assets/52a23923-9f9c-46d8-8a85-2b3cc59c615e)

whereas it should look like so:

> ![CleanShot 2024-11-13 at 21 33 38@2x](https://github.com/user-attachments/assets/8f833412-e9ee-41ca-a4c8-e6f2dcd64ac7)


Source: https://github.com/nightscout/trio-docs/tree/082d0729fe32ba22e622ad0b7abdd2f97917fa94#migrate-an-admonition-from-sphinx-to-mkdocs

Natolida replied:
Thank you very much; I kept making changes to troubleshoot why it wasn't displaying correctly.

I have gone through and made the changes as per the link provided. However, when I build locally, the admonition still displays incorrectly. Has the issue been fixed on your end?

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+When you use Build with Mac: 1. When you first build. 2. Once per year, at the minimum, after that. However, if you decide to use a free Apple Developer account, you must rebuild every 7 days. 
+
+!!! important "Important" 
+_Periodically, developers will release updates to Trio. These updates may include new features or important safety updates. It is recommended that as updates are released, users update their Trio app._
```
4-space indentation is missing.

Natolida replied:
fixed

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+Why? Trio is only as good as the information you give it. Basal, carb ratios, and insulin sensitivity factors form the basis for all computing by Trio's algorithms. If you have given Trio poor-quality information, you will see poor-quality output. This will likely present as unexpected behavior to the everyday user and result in a poor user experience.
+
+!!! important "Important"
+_Before commencing on Trio, users must test and refine their basals, carb ratios, and insulin sensitivity factors._
```
4-space indentation is missing.

Natolida replied:
fixed

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+Yes. Trio does not require internet or cell coverage to work. So long as the Trio user has Bluetooth enabled on the iPhone, the CGM and insulin pump will continue working in conjunction with Trio. 
+
+!!! warning
+_If you have chosen a CGM source that requires the internet (i.e., Nightscout or Dexcom Share), users must maintain an internet connection. If you enter an area without internet, you will lose your CGM readings and Trio will stop looping._
```
4-space indentation is missing.

Natolida replied:
fixed

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+<br>LoopFollow is an open-source, do-it-yourself iPhone app that combines your T1D's display information and alerts into a single, easy-to-use interface. LoopFollow pulls data from your Nightscout or Dexcom Share server. You can set up more than one app on a phone if you follow more than one person with diabetes. Information on creating your LoopFollow app can be found [here](https://www.loopandl ...
+
+!!! warning
+_All options for following your Trio app rely on an active internet connection. If you or the person you are following loses connection to the internet, your LoopFollow app will stop receiving data._
```
4-space indentation is missing

Natolida replied:
fixed

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+You will need to increase this value to allow Trio to start automating your insulin delivery. If you are nervous, you can start by setting this value to your average meal bolus. Then, over time, you can begin to increase the value. 
+
+!!! tip 
+_The default recommendation is “average meal bolus + 3x max hourly basal” when using super micro boluses._
```
4-space indentation is missing

Natolida replied:
fixed

ebouchut commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+<br>To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. Please see the following for instructions on how to build xDrip4iOS yourself [here](../operate/build.md#xdrip4ios-as-cgm-source).  
+
+!!! tip 
+_If you use xDrip with the Dexcom G6/ONE, in Trio select 'Dexcom G6' as the CGM source. Trio will intercept glucose readings directly from the transmitter._
```
4-space indentation is missing.

ebouchut reviewed (commented):
Hi @Natolida 

- There is a broken link (an extra dot).
- The 4-space indentation is missing for each line of the body text of each annotation

Natolida committed:
Update faq.md

Corrects syntax for Tips, Warnings, Etc

Natolida committed:
Update faq.md

Updates syntax of tips, warnings, etc

Natolida committed:
Update faq.md

Natolida committed:
Update faq.md

- Updated admonition syntax
- Fixes broken link

Natolida committed:
Update faq.md

-Updated admonition

ebouchut commented:
Admonitions are now laid out correctly.
Thx @Natolida 

I suggest you add [Libre 2+ as supported CGMs](https://github.com/nightscout/trio-docs/pull/95/files#diff-57250391b535b7f396fede9e05576a59d95a62aac82d314b5ae0d2638c66e0eaR42) (Cf. PR #95)

Natolida committed:
Update faq.md

- Adds Libre 2+ to list of supported CGMs

Natolida commented:
Thanks for your assistance, @ebouchut! 

I also thought of the Libre 2+ today and added it to the list. 

This should be complete now and ready to be merged.

MikePlante1 commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+Trio can’t determine the glucose value when under 40mg/dL (2.2 mmol/L). This is why the CGM apps from Dexcom and Libre are displaying “Low.” A “Low” reading is also sometimes due to a sensor error or a faulty sensor needing replacement. Trio can't safely make a glucose prediction without knowing the glucose value or if it’s rising or falling.
+
+### Why isn't Trio looping or delivering insulin when CGM glucose is 400 mg/dl (22 mmol/l) or higher?
+Trio can’t determine the glucose value when over 400mg/dL (22 mmol/L). This is why the CGM apps from Dexcom and Libre display “High.” A “High” reading is also sometimes due to a sensor error or a faulty sensor needing replacement. Trio can't safely make a glucose prediction without knowing the glucose value or if it’s rising or falling.  
```
*22.2 mmol/L

Natolida replied:
Good catch on that one 😄

MikePlante1 commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+>* US Libre 1 10-day sensors via Transmitters
+>* US Libre 1 14-day sensors via Transmitters
+>* International Libre 1 sensors via Transmitters
+>* European Libre 2/2+ sensors via Transmitters
```
> 2 / 2 Plus

Natolida replied:
Sorted ✔️

MikePlante1 commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+### Does Trio cause an increase in pod failures?
+There is more communication between your phone and the pod than the PDM. This increases the load on the pod battery. Most people have no increase in pod failures. However, it does occasionally happen. Below are some tips to take to limit battery drain:
+* The pod state is updated every 3 minutes for DASH and every 5 minutes for Eros. If uncertain communications are detected such that Trio cannot determine if a dosing command sent to the pod was received, the app will try once a minute to get a response.
+* If using Eros pods with a RileyLink device. Be mindful of situations that may lead to poor signal. Poor communication between devices may lead to battery drain and contribute to faults.
```
Should the first period be a comma?

Natolida replied:
Sorted ✔️

MikePlante1 commented on docs/resources/faq.md:
```diff
@@ -0,0 +1,187 @@
+### What CGMs are supported by Trio?
+The following CGM can be used with Trio:
+* Dexcom G5
+* Dexcom G6/ONE
```
`G6 / ONE`

Natolida replied:
Sorted ✔️

MikePlante1 reviewed (changes requested):
Just had a quick look at this, but I’ll give it a more thorough look later today.

Natolida committed:
Update faq.md

- Fixes Typo
- Grammatical change
- Wording change Libre 2+ --> Libre 2 Plus as per @MikePlante1
- Changes spacing G6/ONE --> G6 / ONE as per  @MikePlante1

Natolida commented:
Thanks for the review, @MikePlante1.

In the previous PR #81 , @tmhastings wanted you to confirm something: Could you check out the information regarding Nightscout as a CGM source to ensure this is still valid or if it was sorted with one of the recent updates?

MikePlante1 committed:
fix bullets in FAQs

bullet lists seem to require an empty line before the first `* `

ebouchut commented:
Closing this PR as it has been superseded by #108.

ebouchut commented:
@tmhastings or @MikePlante1, would one of you close this PR, as I do not have the permission to do so.

MikePlante1 commented:
Closing this PR as it has been superseded by:
* #108
